### PR TITLE
Feature tooltip

### PR DESCRIPTION
**Pull request by shahruzmannan**

### CODE REVIEW

File: client/src/build-view.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ class Build extends React.Component {
               >Back to dashboard
               </button>
             </Link>
-            <label>
+            <label data-tooltip="when the instance has been built the link will work" data-tooltip-width="200px">
```
Comment (HenrikHelenius) [inline client/src/build-view.js]: capitalize

File: client/src/components/comment/comment.js [hunk 2]
```diff
@@ -55,6 +55,7 @@ const devLabel = () => {
   return null
 }
 
+
```
Comment (HenrikHelenius) [inline client/src/components/comment/comment.js]: extra line remove

File: client/src/components/comment/comment.js [hunk 3]
```diff
@@ -65,9 +66,22 @@ const Comment = ({ id, comment, role, op, onClick, buttonText, canDelete, delete
         </div>
         {opLabel(op, comment.userId)}
       </div>
-      <div className={css('time-target-container')}>
-        <Moment className={css('timestamp')} fromNow>{comment.time}</Moment>
-        {targetElement(comment)}
+      <div
+        className={css('time-target-container')}
+        data-tooltip={comment.time}
```
Comment (HenrikHelenius) [inline client/src/components/comment/comment.js]: remove when not correctly formatted

File: client/src/components/comment/comment.scss [hunk 4]
```diff
@@ -1,7 +1,11 @@
+@import '../../scss/variables';
```
Comment (HenrikHelenius) [inline client/src/components/comment/comment.scss]: compulsory has already these, remove extra, see for yourself

File: client/src/components/comment/comment.scss [hunk 5]
```diff
@@ -1,7 +1,11 @@
...
+@import '../../scss/colors';
+@import '../../scss/mixins';
 @import '../../scss/compulsory';
 
 @import '../../scss/atoms-organisms/button';
 
+
```
Comment (HenrikHelenius) [inline client/src/components/comment/comment.scss]: remove extra line

File: client/src/components/submit-field/submit-field.scss [hunk 6]
```diff
@@ -3,6 +3,7 @@
 @import '../../scss/mixins';
 @import '../../scss/atoms-organisms/button';
 
+
```
Comment (HenrikHelenius) [inline client/src/components/submit-field/submit-field.scss:4]: remove extra line

File: client/src/components/submit-field/submit-field.scss [hunk 7]
```diff
@@ -3,6 +3,7 @@
 @import '../../scss/mixins';
```
Comment (HenrikHelenius) [inline client/src/components/submit-field/submit-field.scss]: replace all these with compulsory import

File: client/src/components/tag-element-button/tag-element-button.scss [hunk 8]
```diff
@@ -1,3 +1,6 @@
+@import '../../scss/variables';
```
Comment (HenrikHelenius) [inline client/src/components/tag-element-button/tag-element-button.scss]: remvoe all these as they are in compulsory

File: client/src/create.js [hunk 9]
```diff
@@ -115,7 +115,7 @@ class Create extends React.Component {
             required
           />
         </label>
-        <label htmlFor="port">
+        <label htmlFor="port" data-tooltip="Port number depends on the application type eg. node.js runs on  3000" data-tooltip-width="200px">
```
Comment (HenrikHelenius) [inline client/src/create.js]: break line, too loing

**Review by HenrikHelenius — CHANGES_REQUESTED** [review]

**Review by tobbe-j — DISMISSED** [review]
Looks good

**Review by HenrikHelenius — DISMISSED** [review]

**Review by HenrikHelenius — DISMISSED** [review]

**Review by HenrikHelenius — APPROVED** [review]

**Pull request merged**